svm/src/account_loader.rs

```diff
@@ -234,6 +234,9 @@ fn load_transaction_accounts<CB: TransactionProcessingCallback>(
                 tx_rent += rent;
                 rent_debits.insert(key, rent, account.lamports());
 
+                account
```

Comment:
This particular line should have no effect while feature `deprecate_executable_meta_update_in_bpf_loader` is not activated, because `is_executable(&account, &features_set)` should just return `account.executable` (in `impl ReadableAccount`).

Now I have been running a validator with this particular change and it's working fine. However that's not proof it's not consensus breaking - we should tread carefully.

Can you say any way this could be consensus breaking?

Reply:
Sounds reasonable. 

I am thinking about accounts that owned by native_loader. Will it be possible that executable = false for these accounts? I think not. but just want to make sure!

Reply:
> I am thinking about accounts that owned by native_loader. Will it be possible that executable = false for these accounts? I think not. but just want to make sure!

Thanks for mentioning that - anything that we can check is super valuable, so keep them coming.

However having said that, the way they're loaded in `fn load_and_execute_sanitized_transactions()` (see https://github.com/solana-labs/solana/blob/master/svm/src/transaction_processor.rs#L205 ) means that they're always marked executable. All the other references I could find to the native loader were in tests.